### DISCUSSION TITLE
Help with Launch application API

### DISCUSSION THREAD

**Discussion by tkn1234** [supersaiyanmode/PyWebOSTV#101] [Q&A]
For the class - class ApplicationControl(WebOSControlBase)
I'm trying to start the media player (app id - com.webos.app.mediadiscovery) but I can't find a reference to how to send the destination I want to play (its a network based media player) or what the TV can 'see' etc.

I've found several LG API's, I'm not sure which is the correct one (but they're poorly documented) if you can point me to the right API to model, I can update the code and submit a pull request that enhances that function.

Appreciate any help you can give!  Great library!

**Comment by supersaiyanmode** [comment]
Hi, unfortunately, the "destination" is something that the application must accept as a parameter and honor. It is passed as a part of application launch (see README.md) -- and as such it is outside the scope of this library which doesn't give specific apps any special treatment. If you can figure out the actual parameter, you can pass it via contentId and params arguments of app.launch() -- unless ofcourse these parameters are insufficient and need to change.
